### PR TITLE
feat: Update Orama version and fixed broken links.

### LINKED ISSUE #103: Search hotkey
I was looking to turn on the hotkey: 

image

Would you happen to know how?
(in case we may open a new dedicated issue)

_Originally posted by @Eomm in https://github.com/fastify/website-next/pull/102#discussion_r1225194795_

### PR DESCRIPTION
**Pull request by ShogunPanda**
## Description

This PR updates Orama in order to provide hotkey functionality.
It also refactors broken link handling script and fixes an additional error that raised in docs of version 4.19.

## Related Issues

Fixes #103.

## Check List

- [x] I have read the [Contributing Guidelines](./CONTRIBUTING.md)

### CODE REVIEW

File: scripts/process-releases.js [hunk 1]
```diff
@@ -235,57 +235,74 @@ async function fixHtmlTags(dir) {
...
+    'v4.12.x',
+    'v4.13.x',
+    'v4.14.x',
+    'v4.15.x',
+    'v4.16.x',
+    'v4.17.x',
+    'v4.18.x',
+    'v4.19.x',
+    'latest',
+  ]
```
Comment (mcollina) [inline scripts/process-releases.js]: This doesn't really scale. We should not be changing this file every time we ship a new release of Fastify

**Review by mcollina — CHANGES_REQUESTED** [review]

File: scripts/process-releases.js (hunk 1 above)
Comment (ShogunPanda) [reply scripts/process-releases.js]: I agree. Fun fact, it was unnecessary: it now runs fixing on all directories in the versioned_docs folder.

**Review by mcollina — APPROVED** [review]
lgtm

File: package.json [hunk 2]
```diff
@@ -29,7 +29,7 @@
     "@docusaurus/plugin-client-redirects": "^2.4.1",
     "@docusaurus/preset-classic": "^2.4.1",
     "@mdx-js/react": "^1.6.22",
-    "@orama/plugin-docusaurus": "^1.0.6",
+    "@orama/plugin-docusaurus": "^1.0.8",
```
Comment (Eomm) [inline package.json:32]: Could you commit the package-lock too?

Comment (ShogunPanda) [reply package.json:32]: Done!

**Pull request merged**